packages/plugin-core/src/ExtensionProvider.ts

```diff
@@ -36,6 +36,18 @@ export class ExtensionProvider {
     return ExtensionProvider.getExtension().wsUtils;
   }
 
+  static isActive() {
+    return ExtensionProvider.getExtension().isActive();
+  }
+
+  static getWorkspaceConfig() {
+    return ExtensionProvider.getExtension().getWorkspaceConfig();
+  }
+
+  static getTreeView(key: DendronTreeViewKey) {
```

Comment:
Can we go ahead and slap a `@deprecated` tag on this one? :D 

No issue with adding this in now to get around some circ deps, but going forward with the eventing pattern, we shouldn't need to expose any tree views anymore.

Reply:
sounds good